include/lldb/Symbol/SwiftASTContext.h

```diff
@@ -725,6 +725,8 @@ class SwiftASTContext : public TypeSystem {
                                 lldb::StackFrameWP &stack_frame_wp,
                                 swift::SourceFile *source_file, Status &error);
 
+  static CompilerType GetSwiftCompilerType(swift::Type swift_type);
```

Comment:
isn't the "Swift" in the name redundant?
Should it be `SwiftASTContext::GetCompilerType(...)`?

Reply:
Yeah, it is somewhat redundant. I need to update this PR, I can rename it when I do that.